### PR TITLE
Add Project.toml

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+uuid = "ebbdde9d-f333-5424-9be2-dbf1e9acfb5e"
+authors = ["Vaibhavdixit02 <vaibhavyashdixit@gmail.com>"]
+version = "1.2.0"
+
+[deps]
+ApproxBayes = "f5f396d3-230c-5e07-80e6-9fadf06146cc"
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DynamicHMC = "bbc10e6e-7c05-544b-b16e-64fede858acb"
```
Comment (ChrisRackauckas) [inline Project.toml]: Missing the things like LinearAlgebra and stuff like that. Look in the repo for what you need. Things will throw warnings without it.

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Turing = "fce5fe82-541a-59a6-adf8-730c64b5f9a0"
+DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+LabelledArrays = "2ee39098-c373-598a-b85f-a56591580800"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+LogDensityProblems = "6fdf6af0-433a-55f7-b3ed-c6c6e0b8df7c"
+Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
+PDMats = "90014a1f-27ba-587c-ab20-58faa44d9150"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```
Comment (ChrisRackauckas) [inline Project.toml]: why was this added?

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
```
Comment (ChrisRackauckas) [inline Project.toml]: Why was this added?

File: Project.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+[deps]
+ApproxBayes = "f5f396d3-230c-5e07-80e6-9fadf06146cc"
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DynamicHMC = "bbc10e6e-7c05-544b-b16e-64fede858acb"
+LogDensityProblems = "6fdf6af0-433a-55f7-b3ed-c6c6e0b8df7c"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+Mamba = "5424a776-8be3-5c5b-a13f-3551f69ba0e6"
```
Comment (ChrisRackauckas) [inline Project.toml]: Are we still using Mamba?

Comment (Vaibhavdixit02) [reply Project.toml]: Yes in plotting
